src/mongo_odm/query_builder.hpp

```diff
@@ -292,7 +292,7 @@ class mod_expr {
      * as a key-value pair of the form " key: { $mod: [ divisor, remainder ] } "
 .
      * @param builder a BSON core builder
-     * @param wrap	    Whether to wrap the BSON inside a document.
+     * @param wrap      Whether to wrap the BSON inside a document.
```

Comment:
along as you're doing away with the tabs (here and above), can you fix the alignment?